### DISCUSSION TITLE
Error: P1001 , How can i establish connection

### DISCUSSION THREAD

**Discussion by sidharthhhh** [prisma/prisma#20354] [Q&A]
# Error: P1001: Can't reach database server at `aws.connect.psdb.cloud`:`3306`
# Please make sure your database server is running at `aws.connect.psdb.cloud`:`3306`

This error is comming when i give "npx prisma db push" command in vscode terminal. 
How can i solve this problem? Any suggestion!!

**Comment by ludralph** [comment]
Hi @sidharthhhh 👋 

Could you try connecting with another tool like  DBeaver or TablePlus and see if only Prisma errors or if it's a general problem?

**Reply by sidharthhhh** [reply]
Its connection error ! Still i unable to find its solution

**Reply by ludralph** [reply]
Hi @sidharthhhh, to be clear, Do you still have connection errors when using an external tool like [DBeaver](https://dbeaver.io/) or [TablePlus](https://tableplus.com/)?

**Reply by sidharthhhh** [reply]
it's woking now with planet scale, thanx................
